changelog.txt

```diff
@@ -38,6 +38,7 @@
 * Fix - Translation warning when initializing the status page information.
 * Add - Add ACSS support for WC Subscriptions.
 * Add - Add WooCommerce Pre-Orders support to ACSS.
+* Update - Remove unused shopper tracking
```

Comment:
```suggestion
* Update - Remove unused express checkout button tracking.
```
nitpick: to avoid confusion because we don't have any shopper tracking at all.